dandi/dandiapi.py

```diff
@@ -230,6 +237,17 @@ def __init__(self, api_url=None, token=None):
         if token is not None:
             self.authenticate(token)
 
+    @classmethod
+    def for_dandi_instance(
```

Comment:
`from_` methods remain a bit odd in naming for me, but I think I am growing into "ok"ing having them, although difference from `get_` or `from_` still remains somewhat opaque to me.

Reply:
This is `for_`, not `from_`.